pennylane/qchem/openfermion_obs.py

```diff
@@ -594,19 +596,6 @@ def dipole_of(
     """
     openfermion, _ = _import_of()
 
-    atomic_numbers = {
```

Comment:
Why this is removed?

Reply:
This is now being imported, it was creating a conflict with the imported list, so I imported in both instead of recreating it in multiple places.